docs/source/notes/architecture_overview.rst

```diff
@@ -62,7 +62,7 @@ This is a core and basic feature of NNI, we provide many popular `automatic tuni
 General NAS Framework
```

Comment:
Did we talk about merging `notes` and `misc`?

Reply:
yes, I will merge `notes` and `misc` to `misc`